Guia de boas praticas para mentores(as) (#109)
BOSS-BigOpenSourceSister/BigSister: docs/conteudos/guia_para_mentoras.md

# Guia de boas práticas para mentoras

> Este guia tem o intuito de auxiliar pessoas que participarão do programa como mentores. Para algumas pessoas podem ser a primeira vez que terão a missão de mentorar, e com a experiência do primeira iniciativa BOSS, trazemos aqui dicas de boas práticas. Lembrando que não se trata de regras, obrigatoriedade, mas dicas que te ajudarão a guiar a(o) mentorada(o).

Reviewer: `e com a experiência do primeira iniciativa BOSS`

Seria `da primeira`.

Author: Valeu marina!!! Vou arrumar agora!

- [1:1s](#11s)    
  - [Regular](#regular)     
  - [Pontualidade](#pontualidade)    
  - [Instigue o interesse](#instigue-o-interesse)    
  - [Não há problema em cancelar](#não-há-problema-em-cancelar)    
  - [Documento para acompanhamento](#documento-para-acompanhamento)    
    - [Ferramentas](#ferramentas)    
  - [Dois ouvidos, uma boca](#dois-ouvidos-uma-boca)    
  - [Tópicos para o 1:1](#tópicos-para-o-11)    
  - [Perguntas](#perguntas)    
    - [Principais](#principais)
    - [Construindo relacionamento](#construindo-relacionamento)    
    - [Experiência de trabalho/faculdade](#experiência-de-trabalhofaculdade)    
    - [Comunicação](#comunicação)    
    - [Aprendizagem e crescimento](#aprendizagem-e-crescimento)    
    - [Desenvolvimento de carreira](#desenvolvimento-de-carreira)    
  - [Sinceridade](#sinceridade)    
- [Formulário de acompanhamento participante](#formulário-de-acompanhamento-participante)
- [Formulário de acompanhamento mentor(a)](#formulário-de-acompanhamento-mentora)
- [Referências](#referências)

> ## 1:1s

Reviewer: Acho que aqui vale um tópico `O que é`, pra dar uma visão geral do que é.

Author: pronto @Joranhezon ! corrigido <3

# Guia de boas práticas para mentoras

Este guia tem o intuito de auxiliar pessoas que participarão do programa como mentores. Para algumas pessoas podem ser a primeira vez que terão a missão de mentorar, e com a experiência da primeira iniciativa BOSS, trazemos aqui dicas de boas práticas. Lembrando que não se trata de regras, obrigatoriedade, mas dicas que te ajudarão a guiar a(o) mentorada(o).

> - [1:1s](#11s)    

Reviewer 2: Acho que esse link aqui tá quebrado

  - [Regular](#regular)     
  - [Pontualidade](#pontualidade)    
  - [Instigue o interesse](#instigue-o-interesse)    
  - [Não há problema em cancelar](#não-há-problema-em-cancelar)    
  - [Documento para acompanhamento](#documento-para-acompanhamento)    
    - [Ferramentas](#ferramentas)    
  - [Dois ouvidos, uma boca](#dois-ouvidos-uma-boca)    
  - [Tópicos para o 1:1](#tópicos-para-o-11)    
  - [Perguntas](#perguntas)    
    - [Principais](#principais)
    - [Construindo relacionamento](#construindo-relacionamento)    
    - [Experiência de trabalho/faculdade](#experiência-de-trabalhofaculdade)    
    - [Comunicação](#comunicação)    
    - [Aprendizagem e crescimento](#aprendizagem-e-crescimento)    
    - [Desenvolvimento de carreira](#desenvolvimento-de-carreira)    
  - [Sinceridade](#sinceridade)    
- [Formulário de acompanhamento participante](#formulário-de-acompanhamento-participante)
- [Formulário de acompanhamento mentor(a)](#formulário-de-acompanhamento-mentora)
- [Referências](#referências)

## O que são 1:1s

> 1:1s são encontros onde o(a) mentorado(a) tem com o(a) mentor(a) e que geralmente seguem uma agenda ou tópicos pré definidos para o acompanhento e evolução do(a) mentorado(a). Esses encontros podem ser feitos virtualmente por videoconferência ou pessoalmente e geralmente duram até 1h (não é uma regra). Essa é uma ferramenta de desenvolvimento profissional, onde o(a) mentor(a) é uma pessoa com mais experiência em uma área específica e está disposta a transmitir conhecimento e ajudar o(a) mentorado(a) com possíveis obstáculos. 

Reviewer 2: ```suggestion
1:1s são encontros entre mentora e mentorada (ou big sister e little sister) . Geralmente seguem uma agenda ou tópicos pré definidos para o acompanhento e evolução da pessoa sendo mentorada. Esses encontros podem ser feitos virtualmente por videoconferência ou pessoalmente e geralmente duram até 1h (não é uma regra). 

Essa é uma ferramenta de desenvolvimento profissional, onde a pessoa que mentora é uma pessoa com mais experiência em uma área específica e está disposta a transmitir conhecimento e ajudar a pessoa mentorada com possíveis obstáculos. 
```

### Regular

> É muito importante manter a frequência dos encontros 1:1, pois é um tempo de colaboração para que o(a) mentor(a) se conheçam e se conectem. É um momento de você como mentor(a) ouvir o(a) mentorado(a), para isso você precisa criar esse ambiente utilizando dinâmicas para interagirem (quebra-gelo) e conhecendo um ao outro para que o(a) mentorada(o) sinta liberdade de falar com você.

Reviewer 2: ```suggestion
É muito importante manter a frequência dos encontros 1:1, pois é um tempo de colaboração para que as pessoas envolvidas se conheçam e se conectem. É um momento de você, como mentora, ouvir sua mentorada. Para isso você precisa criar esse ambiente confortável utilizando dinâmicas de interação (quebra-gelo) para que a mentorada sinta liberdade de falar com você.
```

### Pontualidade
Seja pontual nos horários marcados dos 1:1s. É falta de respeito chegar atrasado. O tempo é igual e valioso para todos. Então caso atrase, avise com antecedência ou se for o caso, remarque com antecedência também.

### Instigue o interesse

> Talvez seja a primeira vez de algum(a) mentorado(a), então é legal deixar claro o que é uma mentoria, qual seu papel como mentor(a) e instigar o interesse dele(a). Incentivar que tenham a iniciativa de marcar a mentoria semanal e que traga tópicos/pauta para o 1:1. Caso o(a) mentorado(a) não tome essa iniciativa, você pode puxar ou tentar deixar um horário fixo semanal, mas tente ir instigando essa iniciativa e interesse do mentorado(a). Que é o que acontece geralmente na vida real, pois o interesse é do(a) mentorado(a) em ter um(a) mentor(a).

Reviewer 2: ```suggestion
Talvez seja a primeira vez de alguma mentorada com esse formato. Então é legal deixar claro o que é uma mentoria, qual seu papel como mentora e mostrar que é um espaço que ela pode propor algo. é interessante instigar o interesse dela e incentivar que tenham a iniciativa de marcar a mentoria semanal e que traga tópicos/pauta para o 1:1. 

Caso a mentorada não tome essa iniciativa, você pode puxar ou tentar deixar um horário fixo semanal, mas tente ir instigando essa iniciativa e interesse da mentorada. ``

#### Ferramentas

1. [Notion](https://www.notion.so/)
2. [Google Docs](https://docs.google.com/?hl=pt-BR)

### Dois ouvidos, uma boca
Tem um motivo para você ter dois ouvidos e uma boca. Ouvindo você aprende mais do que falando, então use isso para guiar o 1:1 e ter um resultado mais eficaz. No começo, o(a) mentorado(a) talvez não se sinta muito a vontade de falar, então vá construindo esse ambiente. Talvez para isso você tenha que falar mais até criar uma conexão entre vocês. Use dinâmicas de quebra gelo, autoconhecimento, entre outros.

Este ponto fortalece a iniciativa do(a) mentorado(a) de trazer as pautas. Sugerir ferramentas para que o(a) próprio(a) mentorado(a) possa se organizar com tarefas da semana, objetivos em relação ao programa podendo trazer aos 1:1s e relatando as dificuldades e bloqueios para que você possa ajudar.

Ferramentas que contemplem a metodologia kanban são um exemplo, como: [Trello](https://trello.com/), [Projects Github](https://github.com/features/project-management/)

```

> “Quando você fala, está apenas repetindo aquilo que você já sabe. Mas se você escuta, então pode aprender algo novo." 

Reviewer 2: eu acho uma boa citação. Mas não sei se é uma pessoa que alguém pode se sentir incomodada, por conta dessa fala dele sobre a beleza da mulher importar tanto quanto o cérebro:

> Later in our conversation I also challenged him on another remark he made in 2015, when he said if he was followed by a female Dalai Lama she would have to be attractive.
In another surprise, he reaffirmed his belief that beauty matters as much as brains. "If a female Dalai Lama comes, she should be more attractive," he told me while laughing.

(https://www.bbc.com/news/world-asia-48844825) 

Eu deixaria de fora.

Reviewer: Acho válido deixar de fora.
